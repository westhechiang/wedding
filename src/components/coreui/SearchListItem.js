import styled from 'styled-components';
import { theme } from '../../ui';

const SearchListItem = styled.li`
  width: 100%;
  list-style-type: ${props => (props.bullets ? 'disc' : 'none')};
  margin-top: 5px;
  padding: 20px;
  box-sizing: border-box;
  border-bottom: 1px solid ${theme.color.gray1};

  &:first-child {
    margin-top: 0;
  }

  &:hover {
    cursor: pointer;
  }
`;

export default SearchListItem;
