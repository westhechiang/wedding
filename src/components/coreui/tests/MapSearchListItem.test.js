import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SearchListItem from '../../../components/coreui/SearchListItem';

describe('<SearchListItem />', () => {
  it('should match its empty snapshot', () => {
    const wrapper = shallow(<SearchListItem />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should match its "bullets" snapshot', () => {
    const wrapper = shallow(<SearchListItem bullets />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
