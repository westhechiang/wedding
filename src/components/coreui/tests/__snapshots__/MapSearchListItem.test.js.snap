// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchListItem /> should match its "bullets" snapshot 1`] = `
<li
  className="sc-bdVaJa kBNohT"
/>
`;

exports[`<SearchListItem /> should match its empty snapshot 1`] = `
<li
  className="sc-bdVaJa ivmTcs"
/>
`;
